node/podcontroller.go

```diff
@@ -194,32 +198,18 @@ func NewPodController(cfg PodControllerConfig) (*PodController, error) {
 	if cfg.PodInformer == nil {
 		return nil, errdefs.InvalidInput("missing pod informer")
 	}
-	if cfg.ConfigMapInformer == nil {
```

Comment:
Any reason to switch these to being optional?

Reply:
Gives the provider the option to handle this themselves rather than having it embedded in the runtime.